libvirt/cloudinit_def.go

```diff
@@ -22,6 +23,10 @@ import (
 const USERDATA string = "user-data"
 const METADATA string = "meta-data"
 
+type UserDataStruct struct {
```

Comment:
`SomethingStruct` is probably the worst name you could choose for a `struct`... :-)